source/BoardingPanel.cpp

```diff
@@ -538,17 +538,34 @@ void BoardingPanel::DoKeyboardNavigation(const SDL_Keycode key)
 BoardingPanel::Plunder::Plunder(const string &commodity, int count, int unitValue)
 	: name(commodity), outfit(nullptr), count(count), unitValue(unitValue)
 {
-	UpdateStrings();
+	UpdateStrings(false);
 }
 
+std::string BoardingPanel::GenerateOutfitLootName(const PlayerInfo& player,
+												  const Outfit *outfit, const int count) const
+{
+	std::string label = outfit->DisplayName();
+
+	if(!player.OutfitIsKnown(*outfit))
+	{
+		label = "Unknown " + outfit->Category();
 
+		if(outfit->Category() == "Power")
+			label += " Component";
```

Comment:
I don't know why we would get special treatment for the power category?

Reply:
because Unknown System, Unknown Gun, Unknown Torrent and Unknown Ammunition sounds logical.
Unknown Power sounds like some super hero stuff...
Unknown Power Component sounds logical.

Reply:
that's true but idk if it's a good idea to make fixed exceptions like this

Reply:
thats quite a change, no?

Reply:
Would be for another PR yes

Reply:
ok, will revert and open another PR on that